src/cdk/testing/test-element.ts

```diff
@@ -101,8 +101,12 @@ export interface TestElement {
    */
   sendKeys(modifiers: ModifierKeys, ...keys: (string | TestKey)[]): Promise<void>;
 
-  /** Gets the text from the element. */
-  text(): Promise<string>;
+  /**
+   * Gets the text from the element.
+   * @param options Options that affect what text is included
```

Comment:
I think if we want the documentation for this to show up correctly in IDEs, we need to make an interface for the config with JsDoc for each member

Reply:
You can remove the bullet point for exclude now that its covered in the interface docs